app/renderer/components/common/commonForm.js

```diff
@@ -7,59 +7,39 @@ const ImmutableComponent = require('../immutableComponent')
 
 const {StyleSheet, css} = require('aphrodite/no-important')
 const globalStyles = require('../styles/global')
-const commonStyles = require('../styles/commonStyles')
 
+const FlyoutDialog = require('./flyoutDialog')
 const {FormDropdown} = require('./dropdown')
 const {FormTextbox} = require('./textbox')
 
 /* See about:styles for templates */
 
 class CommonForm extends ImmutableComponent {
   render () {
-    return <div className={css(
-      commonStyles.flyoutDialog,
-      styles.commonForm
-    )} {...this.props} />
-  }
-}
-
-class CommonFormSmall extends ImmutableComponent {
-  render () {
-    return <div className={css(
-      commonStyles.flyoutDialog,
-      styles.commonForm,
-      styles.commonFormSmall
-    )} {...this.props} />
-  }
-}
-
-class CommonFormMedium extends ImmutableComponent {
-  render () {
-    return <div className={css(
-      commonStyles.flyoutDialog,
-      styles.commonForm,
-      styles.commonFormMedium,
-    )} {...this.props} />
-  }
-}
-
-class CommonFormLarge extends ImmutableComponent {
-  render () {
-    return <div className={css(
-      commonStyles.flyoutDialog,
+    return <FlyoutDialog custom={[
       styles.commonForm,
-      styles.commonFormLarge
-    )} {...this.props} />
+      this.props.small && styles.commonForm_small,
+      this.props.medium && styles.commonForm_medium,
+      this.props.large && styles.commonForm_large,
+      this.props.custom
```

Comment:
in a follow-up: these could be described in about styles too

Reply:
Agreed. I'll add some comment to `about:styles#commonForm`